opentech/apply/users/models.py

```diff
@@ -63,15 +46,46 @@ def get_or_create_and_notify(self, defaults=dict(), site=None, **kwargs):
         return user, created
 
 
-class User(AbstractUser):
+class User(AbstractBaseUser, PermissionsMixin):
+    email = models.EmailField(_('email address'), max_length=255, unique=True)
+    full_name = models.CharField(verbose_name='Full name', max_length=255, blank=True)
+    is_staff = models.BooleanField(
+        verbose_name='staff status',
+        default=False,
+        help_text='Designates whether the user can log into this admin site.',
+    )
+    is_active = models.BooleanField(
+        verbose_name='active',
+        default=True,
+        help_text='Designates whether this user should be treated as active. '
+                  'Unselect this instead of deleting accounts.',
+    )
+    date_joined = models.DateTimeField(verbose_name='date joined', default=timezone.now)
+
     USERNAME_FIELD = 'email'
-    email = models.EmailField(_('email address'), unique=True)
     REQUIRED_FIELDS = []
 
     # Remove the username field which is no longer used
     username = None
 
     objects = UserManager()
 
+    class Meta:
+        ordering = ['id']
+
     def __str__(self):
         return self.get_full_name()
+
+    def clean(self):
+        super().clean()
+        self.email = self.__class__.objects.normalize_email(self.email)
```

Comment:
a) You don't need to access this through the class. 

b) Do you need to do this? The email is normalised on `_create_user`.